### PR TITLE
[SecurityBundle] simplified code

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Simplified code for 3.2 (followup to #21679)

### CODE REVIEW

**Comment by xabbuh** [comment]
We need to be explicit about the supported versions (should be `^3.2.2` because of #20995) of the DependencyInjection component.

File: src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php [hunk 1]
```diff
@@ -32,20 +35,7 @@ public function process(ContainerBuilder $container)
...
-        foreach ($container->findTaggedServiceIds('security.voter') as $id => $attributes) {
-            $priority = isset($attributes[0]['priority']) ? $attributes[0]['priority'] : 0;
-            $voters[$priority][] = new Reference($id);
-        }
-
-        krsort($voters);
-        $voters = call_user_func_array('array_merge', $voters);
-
-        if (!$voters) {
-            throw new LogicException('No security voters found. You need to tag at least one with "security.voter"');
```
Comment (stof) [inline src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php]: this exception is gone. Is it expected ?

Comment (fabpot) [inline src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php]: No, that's a mistake.

Comment (fabpot) [inline src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php]: fixed, test added

**Pull request merged**